pkg/helpers/config/gcpUserData.yaml

```diff
@@ -0,0 +1,16 @@
+#cloud-config
+repo_update: true
+package_update: true
+
+packages:
+  - docker 
+
+runcmd:
+  - sudo mkdir  ~/../home/test
+  - echo "${USERDATA_BEGIN}" >> ~/../home/test/userdata-output
+  - sudo docker pull ${VALIDATOR_IMAGE}
+  # Use `|| true` to ignore failure exit codes, we want the script to continue either way
+  # - [ sh, -c, 'sudo docker run --env "AWS_REGION=${AWS_REGION}" -e "START_VERIFIER=${VALIDATOR_START_VERIFIER}" -e "END_VERIFIER=${VALIDATOR_END_VERIFIER}" ${VALIDATOR_IMAGE} --timeout=${TIMEOUT} || true' 
+  - sudo docker run --env "AWS_REGION=${AWS_REGION}" -e "START_VERIFIER=${VALIDATOR_START_VERIFIER}" -e "END_VERIFIER=${VALIDATOR_END_VERIFIER}" ${VALIDATOR_IMAGE} --timeout=${TIMEOUT}  >> ~/../home/test/userdata-output || echo "Failed to successfully run the docker container"
+  - echo "${USERDATA_END}" >> ~/../home/test/userdata-output
+  - cat ~/../home/test/userdata-output #>/dev/console
```

Comment:
Does it really require to be different from the AWS one ?

Reply:
Yes, the original userdata.yaml did not work. There are some directory issues.

Reply:
Update: the default userdata.yaml file is working :)